client/platform/web-girder/views/ViewerLoader.vue

```diff
@@ -84,8 +95,9 @@ export default defineComponent({
 <template>
   <Viewer
     :id="id"
-    :key="id"
+    :key="`${id}/${revisionNum}`"
```

Comment:
I think this is minor.  But if `revisionNum === undefined` the key may seem weird but it should still work (All undefined keys should be the same revision anyways in terms of vDOM)

Reply:
It's just a hashable string, so it works fine like this.   What would you like to change it to?

Reply:
leave it, I guess if it doesn't cause issues it's fine.